src/main/java/bsh/PreparsedScript.java

```diff
@@ -1,12 +1,11 @@
 package bsh;
```

Comment:
I have a feeling if this requires any change outside of `Interpreter` then it is a sign that we are doing something wrong.